etc/config/metadata/runtime-images/anaconda.json

```diff
@@ -2,7 +2,7 @@
   "display_name": "Anaconda (2021.11) with Python 3.x",
   "metadata": {
     "description": "Anaconda 2021.11",
-    "image_name": "continuumio/anaconda3:2021.11",
+    "image_name": "continuumio/anaconda3:a2816acd3acda208d92e0bf6c11eb41fda9009ea20f24e123dbf84bb4bd4c4b8",
```

Comment:
`@sha256:` would be correct. Works with Docker and Kubernetes, too, e.g. Docker on my laptop

```
$ docker pull continuumio/anaconda3@sha256:8115a7841dc0f82e0a9c2d156239164c86f0b85c071d4dc8d957bd8f3d24e024
docker.io/continuumio/anaconda3@sha256:8115a7841dc0f82e0a9c2d156239164c86f0b85c071d4dc8d957bd8f3d24e024: Pulling from continuumio/anaconda3
a603fa5e3b41: Pull complete 
e732803b8b3a: Pull complete 
Digest: sha256:8115a7841dc0f82e0a9c2d156239164c86f0b85c071d4dc8d957bd8f3d24e024
Status: Downloaded newer image for continuumio/anaconda3@sha256:8115a7841dc0f82e0a9c2d156239164c86f0b85c071d4dc8d957bd8f3d24e024
docker.io/continuumio/anaconda3@sha256:8115a7841dc0f82e0a9c2d156239164c86f0b85c071d4dc8d957bd8f3d24e024
```